app/src/test/java/io/github/droidkaigi/confsched2017/util/AssetsUtilTest.kt

```diff
@@ -0,0 +1,47 @@
+package io.github.droidkaigi.confsched2017.util
+
+import com.google.firebase.FirebaseApp
+import com.taroid.knit.should
+import org.junit.Assert.fail
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+import java.lang.reflect.InvocationTargetException
+
+@RunWith(RobolectricTestRunner::class)
+class AssetsUtilTest {
+    @Test
+    @Throws(Exception::class)
+    fun ctor() {
+        try {
+            val ctor = AssetsUtil::class.java.getDeclaredConstructor()
```

Comment:
[imo]
I think constructor test is unnecessary in this util classes.
If others can initiate AssetsUtil.class, there are not any problems because AssetsUtil always don't have states.